pipelines/DRP.yaml

```diff
@@ -58,6 +58,14 @@ tasks:
         # Setting template 'coaddName' is not sufficient to satisfy contract
         connections.coaddExposures: goodSeeingCoadd
         connections.subtractedExposure: goodSeeingDiff_differenceExp
+  transformDiaSourceCat:
+    class: lsst.ap.association.TransformDiaSourceCatalogTask
+    config:
+      connections.coaddName: goodSeeing
+      connections.diaSourceSchema: goodSeeingDiff_diaSrc_schema
+      connections.diaSourceCat: goodSeeingDiff_diaSrc
+      connections.diffIm: goodSeeingDiff_differenceExp
```

Comment:
Other than the template `connections.coaddName: goodSeeing`, the only connection that needs to be explicitly set here is `connections.diffIm` which is *only* to satisfy the contract checker.  I filed DM-30210 for the contract-checker issue.